frontend/package.json

```diff
@@ -36,7 +36,6 @@
     "marked": "^0.8.0",
     "node-sass": "^4.13.0",
     "osmtogeojson": "^3.0.0-beta.3",
-    "react": "^16.13.1",
```

Comment:
I think it's not good to remove react here. We need to control its version.

Reply:
sorry I don't think this was intended. Will fix